### DISCUSSION TITLE
I need to make a query to sum the result of a multiplication

### DISCUSSION THREAD

**Discussion by aecdanjundex** [prisma/prisma#19081] [Q&A] [answered]
I need to make a query to sum the amount * price of list of items.

i need a query like this: 

`SELECT SUM(amount * price) as total FROM ...`

using prisma...

how can i do that using .aggregate(...)?

**Comment by aecdanjundex** [comment]
I saw something similar here: #18093

**Reply by jharrell** [reply, marked as answer]
Hi @aecdanjundex, as mentioned in the issue you linked this is not currently possible in `.aggregate`. You will need to use a [raw query](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access) to accomplish this.
